charts/telegraf-operator/README.md

```diff
@@ -1,17 +1,64 @@
-# Telegraf-operator
+# Telegraf-operator Helm chart
```

Comment:
Prerequisites for this chart? Kubernetes 1.4+? PV support in underlying infrastructure?